dotcom-rendering/src/lib/useBridgetVersion.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react';
+import { environmentClient } from './bridgetApi';
```

Comment:
is the `bridgetApi` file missing from this PR?

Reply:
Good q! We introduced it in https://github.com/guardian/dotcom-rendering/pull/7457

Reply:
It was added in #7457.